Resources/Prototypes/Recipes/Construction/furniture.yml

```
@@ -692,7 +692,7 @@
   targetNode: dresser
   category: construction-category-furniture
   icon:
-    sprite: Structures/Furniture/furniture.rsi
+    sprite: _White/Structures/Furniture/furniture.rsi
```

```suggestion
    sprite: Structures/Furniture/furniture.rsi
```
По пути уже лежит наш комод, все респрайты объектов лучше хранить там же где и исходный объект